hydroflow_plus_test/src/cluster/paxos.rs

```diff
@@ -587,36 +565,21 @@ fn p_p1b<'a, P: PaxosPayload>(
 }
 
 // Proposer logic to calculate the largest ballot received so far.
-#[expect(clippy::type_complexity, reason = "internal paxos code // TODO")]
 fn p_max_ballot<'a, P: PaxosPayload>(
     proposers: &Cluster<'a, Proposer>,
-    a_to_proposers_p1b: Stream<
-        (ClusterId<Acceptor>, P1b<P>),
-        Unbounded,
-        NoTick,
-        Cluster<'a, Proposer>,
-    >,
-    a_to_proposers_p2b: Stream<
-        (ClusterId<Acceptor>, P2b<P>),
-        Unbounded,
-        NoTick,
-        Cluster<'a, Proposer>,
-    >,
+    a_to_proposers_p1b: Stream<P1b<P>, Unbounded, NoTick, Cluster<'a, Proposer>>,
+    a_to_proposers_p2b: Stream<P2b<P>, Unbounded, NoTick, Cluster<'a, Proposer>>,
     p_to_proposers_i_am_leader: Stream<Ballot, Unbounded, NoTick, Cluster<'a, Proposer>>,
 ) -> Singleton<Ballot, Unbounded, NoTick, Cluster<'a, Proposer>> {
-    let p_received_p1b_ballots = a_to_proposers_p1b
-        .clone()
-        .map(q!(|(_, p1b)| p1b.max_ballot));
-    let p_received_p2b_ballots = a_to_proposers_p2b
-        .clone()
-        .map(q!(|(_, p2b)| p2b.max_ballot));
+    let p_received_p1b_ballots = a_to_proposers_p1b.clone().map(q!(|p1b| p1b.max_ballot));
+    let p_received_p2b_ballots = a_to_proposers_p2b.clone().map(q!(|p2b| p2b.ballot));
```

Comment:
This line is no longer useful because acceptors will always just return the ballot that the proposer sent.
The leader preemption logic currently still needs the acceptors to return max_ballot I believe, unless p2b.victory = false can also trigger a new ballot, or we wait for timeouts to expire

Reply:
will fix in follow-up